### PR TITLE
[Maker][CP] - Use blank array param for buzzer

### PR DESCRIPTION
**Pull request by epeach**
Curriculum request to style the block parameters for buzzer.playSong/playNotes. Old notes were displayed in a vertical array to minimize impact of a really long array. This change removes the default values placed in the array. PlayNotes uses an empty array and PlaySong uses an array with spaces - discussion here: https://codedotorg.slack.com/archives/C0T0EGE8L/p1644507117916229

Before / After in Blocks:
Screenshot from 2022-02-16 15-36-48

Before/After in Text:
Screenshot from 2022-02-16 15-40-10

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by maddiedierker** [comment]
just read through the thread and it looks like some updates might be coming through, so i'll hold off until then. in the meantime, it looks like `buzzer.play` is now the only block that uses `stringifySong`: https://github.com/code-dot-org/code-dot-org/blob/e262b1c6e8557d614ecaab49eaadbc8089577797/apps/src/lib/kits/maker/dropletConfig.js#L295-L301

do we want to update/change the default for that block as well?

**Comment by epeach** [comment]
@madelynkasula - I had the same thought and made a ticket to track it! https://codedotorg.atlassian.net/jira/software/c/projects/STAR/boards/26?modal=detail&selectedIssue=STAR-2103

I wanted to split it out because I want to get this change out ASAP for curriculum and want to make sure I'm deprecating carefully. We only use buzzer.play internally and it does the same things as PlaySong, so we should just remove Play (and stringifySong).

File: apps/src/lib/kits/maker/dropletConfig.js [hunk 1]
```diff
@@ -304,14 +302,14 @@ const circuitPlaygroundBlocks = [
...
-    params: [stringifySong(SONG_1D), 120],
+    params: [stringifySong(SONG_SINGLE_NOTE), 120],
     paramButtons: {minArgs: 1, maxArgs: 2}
   },
   {
     func: 'buzzer.playSong',
     category: CIRCUIT_CATEGORY,
     paletteParams: ['notes', 'tempo'],
-    params: [stringifySong(SONG_CHARGE), 120],
+    params: [`[${stringifySong(SONG_CHARGE[0])}]`, 120],
```
Comment (maddiedierker) [inline apps/src/lib/kits/maker/dropletConfig.js:312]: nit: `SONG_CHARGE[0]` is the same as `SONG_SINGLE_NOTE` -- should we use that instead?

**Review by maddiedierker — APPROVED** [review]
left a nit comment/question, but it's not a merge-blocker

File: apps/src/lib/kits/maker/dropletConfig.js (hunk 1 above)
Comment (epeach) [reply apps/src/lib/kits/maker/dropletConfig.js:312]: SONG_CHARGE[0] is ['G3', 0.25] and SONG_SINGLE_NOTE is ['G3']. PlayNotes takes a one dimensional value (note) and PlaySong takes two dimensional (note, length).

Comment (maddiedierker) [reply apps/src/lib/kits/maker/dropletConfig.js:312]: ahh, my bad!

**Pull request merged**